ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -516,6 +517,7 @@
 				CODE_SIGN_STYLE = Manual;
 				CURRENT_PROJECT_VERSION = "$(FLUTTER_BUILD_NUMBER)";
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
+				DEVELOPMENT_TEAM = M88MLNQCU2;
```

Comment:
Should we have our team id in a file that is checked into git?

Reply:
according to the following SO posts it is not a problem: 
https://stackoverflow.com/questions/44417171/should-i-keep-the-apple-developer-team-id-secret and https://stackoverflow.com/questions/38025852/is-the-appid-prefix-sensitive-information 
Without the files used for signing the app it can't be used either.